src/style.css

```diff
@@ -0,0 +1,82 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+.first_bar {
+  position: absolute;
+  width: 900px;
+  height: 70px;
+  left: 48px;
+  top: 71px;
+}
+
+.icon {
+  position: absolute;
+  left: 26px;
+  top: 25px;
+  display: block;
+}
+
+body {
+  font-weight: 300;
+  color: rgba(61, 78, 97, 1);
+}
+
+.container {
+  position: relative;
+  margin-top: 20px;
+}
+
+.search_bar {
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border: 1px solid #e1e7ed;
+  padding: 0;
+  padding-left: 62px;
+  width: 100%;
+  height: 70px;
+  border-radius: 4px;
+  font-size: 16px;
+  font-family: "Avenir", sans-serif;
+}
```

Comment:
let's use this advice https://codeguide.academy/html-css.html#css-order form Ihor